### PR TITLE
docs: reformat exit

### PR DESCRIPTION
**Pull request by Yongxuanzhang**
reformat exit.rst to follow the guide

- [ ] wait for new layout

### CODE REVIEW

File: chapters/cli/exit.rst [hunk 1]
```diff
@@ -1,8 +1,25 @@
+====================
 Gracefully Exit Jina
 ====================
+---------------------------------------------------------------------------------
+Summary: In this section you will learn the ways to shut down Flow and exit Jina.
```
Comment (FionnD) [inline chapters/cli/exit.rst]: Nothing wrong with this, but I think we can use stronger verbs. (https://developers.google.com/tech-writing/one/clear-sentences#choose_strong_verbs). 

What do you think this re-work? 

In this section, you will learn best practices to shut down a Flow and exit Jina correctly.

File: chapters/cli/exit.rst [hunk 2]
```diff
@@ -1,8 +1,25 @@
...
+---------------------------------------------------------------------------------
+
+Feature description and expected outcome
+----------------------------------------
+Jina provide several ways including Python and console to help you exit gracefully.
```
Comment (FionnD) [inline chapters/cli/exit.rst]: I think we can shorten this to: 

There are several methods to exit Jina gracefully. 

Or do you think we are losing important information?

**Review by FionnD — CHANGES_REQUESTED** [review]
Looks good to me :) 

Two minor grammar suggestions. 

Once a Dev has reviewed you can merge

File: chapters/cli/exit.rst (hunk 1 above)
Comment (Yongxuanzhang) [reply chapters/cli/exit.rst]: Thanks! Much better!

File: chapters/cli/exit.rst (hunk 2 above)
Comment (Yongxuanzhang) [reply chapters/cli/exit.rst]: Thanks! Revised in the latest commit

**Review by FionnD — APPROVED** [review]
Looks great to me!

File: chapters/cli/exit.rst [hunk 3]
```diff
@@ -1,8 +1,25 @@
...
+Make sure you install Jina via `Installation <https://docs.jina.ai/chapters/install/os/index.html>`_.
+
+Implementation:
+---------------
 
 In Python
----------
+^^^^^^^^^
 
 If you use ``with`` scope to start a Flow, then all resources (including Pods of all kinds) of the Flow will be released after your move out from the scope.
```
Comment (CatStark) [inline chapters/cli/exit.rst]: Typo in "your" it should be "you" right?

File: chapters/cli/exit.rst [hunk 4]
```diff
@@ -1,8 +1,25 @@
+====================
```
Comment (CatStark) [inline chapters/cli/exit.rst:1]: Could you also add a table of contents? :)
.. contents:: Table of Contents
    :depth: 2

File: chapters/cli/exit.rst (hunk 3 above)
Comment (Yongxuanzhang) [reply chapters/cli/exit.rst]: ✔️

File: chapters/cli/exit.rst (hunk 4 above)
Comment (Yongxuanzhang) [reply chapters/cli/exit.rst:1]: ✔️

**Comment by bwanglzu** [comment]
@Yongxuanzhang do not need to wait for new layout, after being reviewed, merge and we'll re-oranigze it.
